superset/connectors/sqla/models.py

```diff
@@ -308,7 +308,7 @@ def get_time_filter(
         if start_dttm:
             l.append(col >= self.table.text(self.dttm_sql_literal(start_dttm)))
         if end_dttm:
-            l.append(col <= self.table.text(self.dttm_sql_literal(end_dttm)))
+            l.append(col < self.table.text(self.dttm_sql_literal(end_dttm)))
```

Comment:
Yikes! Thanks @zhaoyongjie for catching this. This was exactly what SIP-15 was fixing.